lib/invisible_captcha/view_helpers.rb

```diff
@@ -56,7 +56,13 @@ def visibility_css(css_class, options)
 
       return if visible
 
-      content_tag(:style, media: 'screen') do
+      nonce = if ::Rails::VERSION::STRING >= '5.2'
```

Comment:
maybe we can use the simpler (and already used in other places) statement `Rails.version`? I'd also replace the other occurrences.

Reply:
Sounds good to me! Replaced them all with this snippet: 

`find . -name '*.rb' -print0 | xargs -0 sed -i "" "s/::Rails::VERSION::STRING/Rails.version/g`